### DISCUSSION TITLE
Cara enable ipv6 feat openclash?

### DISCUSSION THREAD

**Discussion by malikshi** [helmiau/HelmiWrt-OS#6] [Q&A]
Bang gmna cara aktifin ipv6 nya biar openwrt+openclash bisa route traffic ipv6 nya.
Kok butuh ipv6? Biar support netflix+ anti captcha google

**Comment by helmiau** [comment]
> Bang gmna cara aktifin ipv6 nya biar openwrt+openclash bisa route traffic ipv6 nya.
> Kok butuh ipv6? Biar support netflix+ anti captcha google

tinggal jalankan **ipv6-helper** di terminal aja bang, fiturnya udah ada dari awal bikin fw dulu, cuma belum dimasukkan ke available command list aja. jadi tetep bisa dijalankan tuh command.

**Reply by malikshi** [reply]
@helmiau error bang
![image](https://user-images.githubusercontent.com/9080737/128631645-2041228c-b1b7-4524-bb58-ebef05a9d91f.png)
 akun trojan support ipv6 buat test di telegram bang.
 sebelumnya pakai fw lain itu ada wan6 nya cuman kagak ada tools openclash/passwall dkk nya maka nya pindah ke fw repo dsini.

**Reply by helmiau** [reply]
gamasalah itu bang, aman, udah siap pakai itu ipv6 nya
